chore: Fixed the README (#18)
deepset-ai/haystack-demos: airgapped-rest_api/README.md

### Docker build process:

For the example here, you need [`docker compose`](https://docs.docker.com/compose/install/) installed on your system.

You’ll also need to clone the `haystack-demos` repository and run the following commands.

1. Read the `docker-compose.yml` , `Dockerfile` , and `retriever-reader.yml` files carefully.

> 2. Make appropriate changes such as choose the required pipeline/component, add/remove unnecessary commands from `Dockerfile`.

Reviewer: ```suggestion
2. Make appropriate changes: choose the desired pipeline and components, add or remove unnecessary commands from the `Dockerfile`.
```

Author: Done! :tada:
